integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -44,7 +46,7 @@ object BasicTests extends TestSuite{
           |cd! 'repl/'src
           |@
           |println(wd relativeTo x)""".stripMargin
-      )
+        )
```

Comment:
Try not to re-format irrelevant things as part of this diff. Things like [this](https://github.com/lihaoyi/Ammonite/pull/403/files#diff-2acce48a0e2b7d464bb53c0acc47e29fR81), and [this](https://github.com/lihaoyi/Ammonite/pull/403/files#diff-bcdb6a0282e047eba770bc309743e114R239) (adding indentation to the whole for-comprehension) may or may not be the "right" formatting, but they have no place in an already-very-large diff like this one. Given that this diff is already large and hard to review for correctness, you should aim to avoid all this sort of minor/irrelevant changes so we can focus on the script-level-caching. If we care enough, we can put these changes into a separate diff later.